tools/utils/README.md

```diff
@@ -0,0 +1,4 @@
+This library contains the source for the Utils like AppxPackaging, AppxPackagingInterop,
+CommandLine, IO, Logger, ProcessRunner, and StandardInput. This library will be
+published as a Nuget package and consumed by sources in other repositories like
+MsixPackagingTool.
```

Comment:
msixpackagingtool isnt public.

None of the consumers today are public except winget cli i believe.

I would leave this very unspecific